include/mega/megaclient.h

```diff
@@ -178,6 +181,31 @@ class MEGA_API FetchNodesStats
     dstime timeToTransfersResumed;
 };
 
+
+// Helper class for MegaClient.  Executes independent tasks on a separate thread or threads.
+// The number of threads can be 0 (eg. for helper MegaApi) in which case something queued is 
+// immediately executed synchronously on the caller's thread
+// Queueing a nullptr instead of a function to execute will cause the threads to exit.
```

Comment:
Is that really necessary? IMHO the client can just call the destructor of this class to tear down all threads.

Reply:
It's an implementation detail so no need to mention it here really - I've moved that comment to the relevant function